tests/auth/feed-page.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect, test } from "@playwright/test";
+
+
+test("should redirect home page to feed page when user is logged in", async ({ page }) => {
+  await page.goto("/");
```

Comment:
if the loggedIn user is not following anyone, then the redirection would be to the /feed/new page.  Well, the test user is following other users already. the check is toHave, is this covering both cases?

Reply:
yes, it would cover both cases but I improved the regex.